source/m2minterfaceimpl.cpp

```diff
@@ -61,7 +61,8 @@ M2MInterfaceImpl::M2MInterfaceImpl(M2MInterfaceObserver& observer,
   _security(NULL),
   _retry_count(0),
   _reconnecting(false),
-  _retry_timer_expired(false)
+  _retry_timer_expired(false),
+  _bootstrapped(true)
```

Comment:
It's set as true to make it work with connector only case where bootstrap is not performed at all. Maybe the flag name can be changed if needed.